src/VisualStudio/Core/Def/LanguageClient/LogHubLspLogger.cs

```diff
@@ -37,7 +37,13 @@ public void Dispose()
         }
 
         public void TraceInformation(string message)
-            => _traceSource.TraceInformation(message);
+        {
+            // Explicitly call TraceEvent here instead of TraceInformation.
+            // TraceInformation indirectly calls string.Format which throws if the message
+            // has unescaped curlies in it (can be a part of a URI for example).
+            // Since we have no need to call string.Format here, we don't.
+            _traceSource.TraceEvent(TraceEventType.Information, id: 0, message);
```

Comment:
Can we add a test?

Reply:
@jasonmalinowski I tried - but I couldn't get it to repro in our unit tests.  It only repro'd under the debugger in VS.  It might have something to do with the version of .netframework we're running in our tests